app/main/menu.js

```diff
@@ -366,9 +378,19 @@ class AppMenu {
 		}];
 	}
 
+	static doNotDisturb() {
+		const newValue = !((ConfigUtil.getConfigItem('showNotification', true) ||
```

Comment:
I don't think this is the right approch here the whole conditional on one line does seem like a good idea its very bug prone. 

Can you instead put the list of all the setting in and array like `var settingList  = ['showNotification', ..]` and then loop through it using `.map` in one liner? 
(Do let me if this is unclear)